src/codemodder/codemods/base_codemod.py

```diff
@@ -162,6 +162,10 @@ def _apply(
             else None
         )
 
+        if results is not None and not results:
```

Comment:
maybe I'm missing something super subtle but can't you just do `if not results`? since results could be None or an empty dict?